app/views/play/level/PlayLevelView.coffee

```diff
@@ -192,6 +192,11 @@ module.exports = class PlayLevelView extends RootView
     @levelLoader = new LevelLoader levelLoaderOptions
     @listenToOnce @levelLoader, 'world-necessities-loaded', @onWorldNecessitiesLoaded
     @listenTo @levelLoader, 'world-necessity-load-failed', @onWorldNecessityLoadFailed
+    if @courseInstanceID
+      fetchACE = $.get("/db/course_instance/#{@courseInstanceID}/classroom?project=aceConfig,members")
```

Comment:
we need to project `members` otherwise we need to update server code. since if we do not project `members` , the mongo won't return `members` then we can't check if the users is a member of classroom 😂 

i prefer `$.get` since it is quite fit our `superModel` 😂  fetchJson somehow works strange. and, `$.get` doesn't need extra require anyway 😂